src/containers/sprite-selector-item.jsx

```diff
@@ -44,18 +54,20 @@ class SpriteSelectorItem extends React.Component {
         // Avoid parsing the SVG when possible, since it's expensive.
         if (asset.assetType === storage.AssetType.ImageVector) {
             // If the asset ID has not changed, no need to re-parse
-            if (this.svgRendererAssetId === this.props.assetId) {
+            if (this.decodedAssetId === this.props.assetId) {
+                // @todo consider caching more than one URL.
```

Comment:
This might be easy with something like https://www.npmjs.com/package/lru-cache

Reply:
Ray was arguing that even the current amount of caching might be premature optimization, so we would have to discuss with him

Reply:
Ah, interesting. Is the idea that if we can avoid calling `decodeText` every frame by some other means then caching here might be unnecessary? Should we meet about this or just chat informally?

Reply:
Ray's argument was ideally sprite-selector-item in its shouldComponentUpdate would only update when assetId changed, which would make the optimization here unnecessary.

However, sprite-selector-item listens on quite a few props, and it's easy to add props without realizing that they might be causing very expensive rerenders (such as dragPayload here), so I think checking the case where the assetID hasn't changed here is worthwhile.

I think that "premature optimization" is only really bad when it's a waste of time or overhead, and this change is small (so far), also here it's not premature because it can cause this big performance issue. I think Ray let it go, possibly because he wanted to go home, but we could all discuss it again if you think that would be good.